sql/world/updates/20251108-00_creature_spawns.sql

```diff
@@ -0,0 +1,34 @@
+-- double spawn
+DELETE FROM `creature_spawns` WHERE `id`=5403;
+DELETE FROM `creature_spawns` WHERE `id`=20663;
+DELETE FROM `creature_spawns` WHERE `id`=34043;
+DELETE FROM `creature_spawns` WHERE `id`=45483;
+DELETE FROM `creature_spawns` WHERE `id`=51331;
```

Comment:
I tried .gocreature 51331 and there is no double spawn on clean OneDB?

Reply:
ok
<img width="839" height="695" alt="image" src="https://github.com/user-attachments/assets/9e71ceb7-10b6-4a97-9787-6695e1967599" />

<img width="708" height="347" alt="image" src="https://github.com/user-attachments/assets/adbd88d7-7076-4963-bf08-d1af5aae3b15" />

<img width="533" height="529" alt="image" src="https://github.com/user-attachments/assets/bc4eccaf-83b2-4637-9dbc-6b9203692144" />